core-sdk-samples/higgs-shop-sample-app/src/features/OrderDetails/OrderDetailsCustomerDetails.tsx

```diff
@@ -23,33 +23,17 @@ const OrderDetailsCustomerDetails: React.FC = () => {
                         id='streetAddress'
                         label='Street Address'
                         value='1234 Normal Lane'
-                        disabled
```

Comment:
If the intent is for someone to be able to customize each of the fields on this page, you'll need to do some sort of handleChange handler on each.  The text boxes are no longer greyed out to make it look like you can edit them but a user actually can't change anything. Is that the intent of this change?

Reply:
Yes. The intent is for them not to appear disabled but to actually be disabled. The intent is to prevent any personally identifiable information to be submitted accidentally.